packages/terra-clinical-item-collection/src/Item.jsx

```diff
@@ -0,0 +1,151 @@
+   * set to 0, and onClick and onKeyDown callbacks set to the onSelect function provided via Item Collection.
+   */
+  isSelectable: PropTypes.bool,
+  /*
+   * Whether or not the item is selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * The view in which the item should be presented. Options are list or table and this will be set by the Item Collection component.
+   */
+  view: PropTypes.oneOf(['list', 'table']),
+};
+
+const defaultProps = {
+  startAccessory: undefined,
+  comment: undefined,
+  endAccessory: undefined,
+  listItemLayout: 'oneColumn',
+  listItemTextEmphasis: 'default',
+  isListItemTruncated: false,
+  accessoryAlignment: 'alignCenter',
+  isSelectable: false,
+  isSelected: false,
+  view: 'list',
+};
+
+function createListItem(elements, selectableProps, isSelected, itemViewStyles) {
+  const { startAccessory, children, comment, endAccessory, hasStartAccessory, hasEndAccessory, ...listItemProps } = elements;
+
+  const listItemContent = (
+    <ItemView
+      startAccessory={startAccessory}
+      displays={React.Children.toArray(children)}
+      comment={comment}
+      endAccessory={endAccessory}
+      reserveStartAccessorySpace={hasStartAccessory}
+      {...itemViewStyles}
+    />
+  );
+
+  return (
+    <List.Item
+      content={listItemContent}
+      isSelected={isSelected}
+      {...selectableProps}
+      {...listItemProps}
+    />
+  );
+}
+
+function createTableCell(content, keyValue, contentType, accessoryAlignment) {
+  const cellClassNames = cx(
+    `content-${contentType}`,
+    { 'content-accessory-align-center': (contentType.includes('accessory') && accessoryAlignment === 'alignCenter') },
+    { 'content-accessory-align-top': (contentType.includes('accessory') && accessoryAlignment === 'alignTop') },
+  );
+
+  return (<Table.Cell content={content} key={keyValue} className={cellClassNames} />);
+}
+
+function createTableRow(elements, selectableProps, isSelected, accessoryAlignment) {
+  const { startAccessory, children, comment, endAccessory, ...tableRowProps } = elements;
+
+  const displayContent = React.Children.map(children, (display, index) => {
+    const displayKey = `display_${index + 1}`;
+    return createTableCell(display, displayKey, 'display');
+  });
+
+  return (
+    <Table.Row isSelected={isSelected} {...selectableProps} {...tableRowProps}>
+      {startAccessory && createTableCell(startAccessory, 'start_accessory', 'start-accessory', accessoryAlignment)}
+      {displayContent}
+      {comment && createTableCell(comment, 'comment', 'comment')}
+      {endAccessory && createTableCell(endAccessory, 'end_accessory', 'end-accessory', accessoryAlignment)}
+    </Table.Row>
+  );
+}
+
+const Item = (props) => {
+  const {
+    view,
+    isSelectable,
+    isSelected,
+    listItemLayout,
+    listItemTextEmphasis,
+    isListItemTruncated,
+    accessoryAlignment,
```

Comment:
nil:  could list these in alphabetical order to match up with the declaration of them at the top